### PR TITLE
feat: add verifier to project page

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: regen-network/regen-registry#1600

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### How to test

https://deploy-preview-1915--regen-registry.netlify.app/project/C45-008

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/components/templates/ProjectDetails/ProjectDetails.utils.tsx [hunk 1]
```diff
@@ -210,12 +211,9 @@ export function getDisplayParty(
   metadata?: AnchoredProjectMetadataBaseLD,
   party?: Maybe<PartyFieldsFragment>,
 ): Party | undefined {
-  const showOnProjectPage = metadata?.[role]?.['regen:showOnProjectPage'];
```
Comment (blushi) [inline web-registry/src/components/templates/ProjectDetails/ProjectDetails.utils.tsx:213]: this was actually from some legacy logic, I believe now the logic with public profile and the latest version of the roles form is rather to display the project stakeholder on the project page as soon as it's defined
cc/ @erikalogie

**Comment by blushi** [comment]
needs https://github.com/regen-network/registry-server/pull/324 before it can be tested since `verifier_id` has been added there to `project` table

**Comment by blushi** [comment]
@erikalogie you can see this on https://deploy-preview-1915--regen-registry.netlify.app/project/C45-008

**Review by wgwz — APPROVED** [review]
utACK

**Comment by erikalogie** [comment]
> > @erikalogie you can see this on https://deploy-preview-1915--regen-registry.netlify.app/project/C45-008
> 
> does this look good? can we merge this?

Sorry I missed this! Yes, it looks good!

**Pull request merged**